kubernetes/samples/scripts/create-rcu-credentials/create-rcu-credentials.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Description
+#  This sample script creates a Kubernetes secret for WebLogic domain admin credentials.
```

Comment:
The description should be for creating the RCU creds not the domain admin...

Reply:
fixed